lib/src/widgets/utils/ds_card.widget.dart

```diff
@@ -240,4 +245,38 @@ class DSCard extends StatelessWidget {
       );
     }
   }
+
+  Widget _buildRequestLocation() {
+    final label = content['label'];
+    final type = label['type'];
+    final value = label['value'];
+
+    return Row(
+      children: DSBubbleUtils.addSpacer(
+        align: align,
+        child: Column(
+          crossAxisAlignment: align == DSAlign.right
+              ? CrossAxisAlignment.end
+              : CrossAxisAlignment.start,
+          children: [
+            if (label != null && type == DSMessageContentType.textPlain)
+              Padding(
+                padding: const EdgeInsets.only(
+                  bottom: 3.0,
+                ),
+                child: DSTextMessageBubble(
+                  text: value,
+                  align: align,
+                  borderRadius: borderRadius,
+                  style: style,
+                ),
+              ),
+            DSRequestLocationButton(
```

Comment:
For now, the `RequestLocationButton` does not need to be visible. I'd like to suggest you to check the request location feature on desk web and blip chat.